preview/scripts/edit-description.js

```diff
@@ -15,12 +15,20 @@ module.exports = async ({inputs, github, context}) => {
     } else {
         // Log warning here?
     }
-    const RTD_PROJECT_DOMAIN = `https://${RTD_PROJECT_SLUG}--${PR_NUMBER}.${RTD_DOMAIN}/`;
 
-    if (RTD_SINGLE_VERSION === "true") {
-        RTD_URL = RTD_PROJECT_DOMAIN;
-    } else {
-        RTD_URL = RTD_PROJECT_DOMAIN + `${RTD_PROJECT_LANGUAGE}/${PR_NUMBER}/`;
+    let RTD_URLS = [];
+
+    for (let i = 0; i < RTD_PROJECT_LANGUAGES.length; i++) {
+        const RTD_PROJECT_LANGUAGE = RTD_PROJECT_LANGUAGES[i];
+        const RTD_PROJECT_DOMAIN = `https://${RTD_PROJECT_SLUG}--${PR_NUMBER}.${RTD_DOMAIN}/`;
+
+        if (RTD_SINGLE_VERSION === "true") {
+            RTD_URL = RTD_PROJECT_DOMAIN;
+        } else {
+            RTD_URL = RTD_PROJECT_DOMAIN + `${RTD_PROJECT_LANGUAGE}/${PR_NUMBER}/`;
+        }
```

Comment:
This could be checked only once --at project level--, since multi-language and single versions are incompatible.